platform/viewer/public/config/default.js

```diff
@@ -25,6 +25,16 @@ window.config = {
   },
   // filterQueryParam: false,
   defaultDataSourceName: 'dicomweb',
+  /* Dynamic config allows user to pass "configUrl" query string this allows to load config without recompiling application. The regex will ensure valid configuration source */
+  dangerouslyUseDynamicConfig: {
```

Comment:
You can leave the configuration commented out with the enabled: true set to show how to enable it.  That way the entire block just works if someone uncomments it.

Reply:
I agree @wayfarer3130 .